src/appsignal/config.py

```diff
@@ -116,11 +125,11 @@ def option(self, option: str) -> Any:
         return self.options.get(option)
 
     @classmethod
-    def load_from_system(_cls) -> Options:
```

Comment:
We can't prefix it with an underscore, even if we don't use it?

Reply:
Yes. The convention is to always use `cls` for the first argument of classmethods. If you don't need the class in the method, use `@staticmethod` instead.

Reply:
Ah, then these should either be in the `Options` object (and use `cls` to instantiate the class) or be modified to use `@staticmethod`. We can change that in a separate PR.

Reply:
Yes, let's make it a static method then. It doesn't need to know the class's state.